mainpage.dox

```diff
@@ -1955,6 +1956,51 @@ var websocket = new WebSocket('ws://1.2.3.4:8188', 'janus-protocol');
 	}
 }
 \endverbatim
+ *
+ * \section signed HMAC-Signed token authentication
+ * Simple token based authentication requires the application host to
+ * continuously update the Janus instance on permission changes.
+ * Since Janus stores the tokens in memory, it can be problematic to guarantee
+ * the permissions of a dynamic application stay in sync with Janus.
+ *
+ * This problem can be solved by using a type of nonce / lease system to
+ * let the application server generate automatically expiring tokens without
+ * requiring direct communication with or any data storage in Janus.
+ *
+ * You can use the HMAC signed token mechanism by enabling token authentication
+ * in general, as above (\c -A or \c --token-auth) and specifying an encryption
+ * secret using \c --token-auth-secret. The same can be accomplished using
+ * \c token_auth and \c token_auth_secret in the \c general section of
+ * \c janus.cfg.
+ *
+ * With Signed token support enabled, dynamic token creation via the \ref admin
```

Comment:
This reminds me that this PR should also partially rename some UI elements in admin.html/admin.js (and probably rewrite a couple of lines of text), as we have a tab there to create/destroy tokens. Pasting the lines I think are relevant just for making this easier:

https://github.com/meetecho/janus-gateway/blob/master/html/admin.html#L45
https://github.com/meetecho/janus-gateway/blob/master/html/admin.html#L72
https://github.com/meetecho/janus-gateway/blob/master/html/admin.html#L138
https://github.com/meetecho/janus-gateway/blob/master/html/admin.js#L954